build_integration_test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+cli="./faas-cli"
+if [ $(uname) == 'Darwin' ] ; then
```

Comment:
What would happen on an ARMHF platform for this test?

Reply:
it's updated on the next lines